common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -788,7 +788,8 @@ def _load_items(self, course_entry, block_keys, depth=0, **kwargs):
         if should_cache_items:
             self.cache_items(runtime, block_keys, course_entry.course_key, depth, lazy)
 
-        return [runtime.load_item(block_key, course_entry, **kwargs) for block_key in block_keys]
+        with self.bulk_operations(course_entry.course_key, emit_signals=False):
+            return [runtime.load_item(block_key, course_entry, **kwargs) for block_key in block_keys]
```

Comment:
`self.cache_items()`, defined above in this same file, includes a `bulk_operations` context manager, but `runtime.load_item` does no such thing, despite (eventually) making use of split.py's `get_cached_block()` and `cache_block()` methods which touch the problematic dictionary.

I elected to keep the context manager usage in split.py; It already does that for other calls so it seems logical to keep it here.